### PR TITLE
Fix Code Formatting in Composer PR

### PR DESCRIPTION
**Pull request by emdashcodes**
### What/Why

This PR follows up on https://github.com/WordPress/tumblr-importer/pull/15 and runs the linter against the changes. The files were using spaces instead of tabs. Most of these are auto fixed.

I also added a `.editorconfig` to help us stay consistent and some GitHub workflows to help catch things during the PR process.

### Testing
* Do some smoke testing, though most of these are space -> tab changes

### CODE REVIEW

**Comment by rtio** [comment]
for som reason it didn't change the base automatically to trunk so I've done that manually. fyi: I've squashed the previous pr so some conflicts have to be fixed

File: .editorconfig
Comment (rtio) [inline .editorconfig:1]: nice

File: .github/workflows/php-coding-standards.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  push:
+    branches: [ trunk ]
+  pull_request:
+    branches: [ trunk ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        php: [ 8.2 ]
```
Comment (rtio) [inline .github/workflows/php-coding-standards.yml]: php 8.2 is reaching end of active support should we use 8.3 here?
image

File: .github/workflows/php-syntax-errors.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  push:
+    branches: [ trunk, develop ]
+  pull_request:
+    branches: [ trunk, develop ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        php: [ 5.6, 7.0, 7.1, 7.2, 7.3, 7.4, 8.0, 8.1, 8.2, 8.3 ] # Include all PHP versions between the absolute minimum that shouldn't throw an error and the latest release version.
```
Comment (rtio) [inline .github/workflows/php-syntax-errors.yml]: let's also include 8.4 here

File: .github/workflows/php-coding-standards.yml (hunk 1 above)
Comment (emdashcodes) [reply .github/workflows/php-coding-standards.yml]: Done!

File: .github/workflows/php-syntax-errors.yml (hunk 2 above)
Comment (emdashcodes) [reply .github/workflows/php-syntax-errors.yml]: Done!

**Comment by emdashcodes** [comment]
@rtio I have rebased this, updated the PHP versions, and made a couple additional fixes to make sure all the new checks pass ✅

**Review by rtio — APPROVED** [review]
:shipit:

**Pull request merged**